src/components/elements/table/RestockPurchasesTable.tsx

```diff
@@ -0,0 +1,220 @@
+import React, {useEffect} from 'react';
+
+import Paper from '@material-ui/core/Paper';
+import {createStyles, makeStyles, Theme} from '@material-ui/core/styles';
+import Table from '@material-ui/core/Table';
+import TableBody from '@material-ui/core/TableBody';
+import TableCell from '@material-ui/core/TableCell';
+import TableContainer from '@material-ui/core/TableContainer';
+import TableHead from '@material-ui/core/TableHead';
+import TablePagination from '@material-ui/core/TablePagination';
+import TableRow from '@material-ui/core/TableRow';
+import TableSortLabel from '@material-ui/core/TableSortLabel';
+
+import {RestockDataEntry} from '../../../state/restock/restockData';
+import {RestockStatusType} from '../../../state/restock/restockData';
+import {restockDispatchers} from '../../../state/restock/restockDispatchers';
+import {useRestockSelector} from '../../../state/restock/restockSelector';
+import {useDispatch} from '../../../state/store';
+import {stableSort, getComparator, Order} from '../../../utils/Sort';
+import {NoData} from './NoData';
+import PurchaseModal from './PurchaseModal';
+import Status from './Status';
+
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    root: {
+      width: '100%',
+    },
+    paper: {
+      width: '100%',
+      marginBottom: theme.spacing(2),
+    },
+    table: {
+      minWidth: 750,
+    },
+    visuallyHidden: {
```

Comment:
Not sure if this is needed, remove it if it's for debugging purposes only.

Reply:
that's needed.